var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -11,10 +11,16 @@ class Gaudi(CMakePackage):
 
     homepage = "http://gaudi.web.cern.ch/gaudi/"
     git      = "https://gitlab.cern.ch/gaudi/Gaudi.git"
+    url      = "https://gitlab.cern.ch/gaudi/Gaudi/-/archive/v33r1/Gaudi-v33r1.tar.gz"
 
-    version('develop', branch='master')
-    version('30.5',    commit='2c70e73ee5b543b26197b90dd59ea4e4d359d230')
-    version('31.0',    commit='aeb156f0c40571b5753a9e1dab31e331491b2f3e')
+    version('master', branch='master')
+    # major cmake config overhaul already in use by some
+    version('develop', git='https://gitlab.cern.ch/clemenci/Gaudi.git', branch='cmake-modernisation')
+    version('33.1', sha256='7eb6b2af64aeb965228d4b6ea66c7f9f57f832f93d5b8ad55c9105235af5b042')
+    version('33.0', sha256='76a967c41f579acc432593d498875dd4dc1f8afd5061e692741a355a9cf233c8')
+    version('32.2', sha256='e9ef3eb57fd9ac7b9d5647e278a84b2e6263f29f0b14dbe1321667d44d969d2e')
+
+    maintainers = ['drbenmorgan', "vvolkl"]
 
     variant('tests', default=False,
             description='Prepare to run the test suite')
```

Comment:
I would remove this variant, you can check to see if tests are requested via `self.run_tests` and use the `type='test'` dependency type.

Reply:
Hmmm, I'm not sure in this case. The `GaudiExamples` subdirectory won't build if `~tests` and while it is not needed for most applications, it contains some 'components' that are loadable at runtime. So in my mind it would be good to be able to tell from the spec if it was built. I could maybe add it to the variant `optional`?

Reply:
In that case, if it actually changes what gets installed, then a variant is fine

Reply:
Ok, I finally used `+optional` for `GaudiExamples` as well and removed `+tests`